### PR TITLE
Minor changes to correct grammatical errors and awkward wording

### PR DESCRIPTION
**Pull request by nlharris**
These minor changes should not affect the meaning.

### CODE REVIEW

File: code-of-conduct/README.md [hunk 1]
```diff
@@ -21,4 +21,4 @@
...
 In 2018 and 2019, The Carpentries Code of Conduct was revised to add a summary, straightforward examples of both beneficial and unwanted behaviors, evaluating intent, reporting guidelines, an appeal process, the procedure for following up with a reportee, terminology, Code of Conduct incident response procedure, termed suspension checklist, expanded clauses for conflicts of interest, and committee membership agreement -- all of which have been adopted by the OBF Code of Conduct. Contributors of these revised documents are Ethan White, Kari L. Jordan, Karin Lagesen, Malvika Sharan, Samantha Ahern, and Simon Waldman.
 
-OBF-specific modifications were added by Malvika Sharan, Yo Yehudi, Bastian Greshake Tzovaras, and Code of Conduct specific emails were created by Hilmar Lapp.
+OBF-specific modifications were added by Malvika Sharan, Yo Yehudi, and Bastian Greshake Tzovaras, and Code of Conduct specific emails were created by Hilmar Lapp.
```
Comment (peterjc) [inline code-of-conduct/README.md:24]: emails -> email addresses?

**Review by peterjc — APPROVED** [review]

**Review by hzahroh — APPROVED** [review]

**Review by hlapp — APPROVED** [review]

File: code-of-conduct/README.md (hunk 1 above)
Comment (nlharris) [reply code-of-conduct/README.md:24]: I wasn't sure whether it meant email addresses, or that he put together announcements that were sent by email! If the latter, then yes, we should correct as you suggested. But I'm wondering why such acknowledgement is even necessary (other than for our own internal reference). I do little administrative things like setting up mailing lists every day and I never get public acknowledgement!

**Comment by nlharris** [comment]
I need one more approval! @HLWiencko @kipkurui @cjfields @gedankenstuecke

I can't @ iddo because he's not a member of https://github.com/orgs/OBF/teams/board, and I don't seem to have admin rights to add him there.

**Comment by peterjc** [comment]
I've invited @idoerg to the board team, that's was probably an oversight in our on-boarding process

**Review by kipkurui — APPROVED** [review]

**Pull request merged**